src/containers/General/Main.tsx

```diff
@@ -1,13 +1,44 @@
 import React from 'react'
+import Voice from 'react-native-voice'
 
-import Screen from '../../screens/General/Main'
+import { Button } from 'react-native'
 
 export default class MainContainer extends React.Component {
+  constructor(props: object) {
+    super(props)
+    Voice.onSpeechResults = this.onSpeechResultsHandler.bind(this)
+  }
+
+  componentWillUnmount() {
+    Voice.destroy().then(Voice.removeAllListeners)
+  }
+
   render() {
-    return <Screen />
+    return (
+      <>
+        <Button onPress={this.onStartButtonPress} title="Start recognize" />
+        <Button onPress={this.onStopPress} title="Stop recognize" />
+      </>
+    )
+  }
+
+  private async onStartButtonPress() {
+    try {
+      await Voice.start('en-US')
+    } catch (e) {
+      console.log(e, 'onStartButtonPressErr')
+    }
+  }
+
+  private async onStopPress() {
+    try {
+      await Voice.stop()
```

Comment:
Cool, but how does it help us to even test this feature? Is it working? Output the result somewhere at least. And make it relevant to what you are doing within this project.

Reply:
I've just updated for testing

Reply:
?